tls/ownership/ownership.md

```diff
@@ -98,6 +100,35 @@
       **Description:** 
       
 
+      Other locations:
+
+      * file /etc/docker/certs.d/virthost.ostest.test.metalkube.org:5000/ca.crt
+      
+
+
+
+## Image Registry (2)
+### Certificate Authority Bundles (2)
+1. file /etc/docker/certs.d/image-registry.openshift-image-registry.svc.cluster.local:5000/ca.crt
```

Comment:
These files both have the same owner and the same content, so they are printed twice here. Not sure if its worth fixing tbh

Reply:
> These files both have the same owner and the same content, so they are printed twice here. Not sure if its worth fixing tbh

Listing every file, including those with duplicate content, is the proper thing to do.  Think about when descriptions are added and those descriptions indicate which binaries and which flags for those binaries the file is used for.  We need those descriptions to be unique.  Even if this specific case doesn't need it, the general case does.